test/test_multi_region_cubed_sphere.jl

```diff
@@ -92,29 +92,63 @@ function get_boundary_indices(Nx, Ny, Hx, Hy, ::North; operation=nothing, index=
     return range_x, Ny-Hy+1:Ny
 end
 
+# Solid body rotation
+R = 1        # sphere's radius
+U = 1        # velocity scale
+φʳ = 0       # Latitude pierced by the axis of rotation
+α  = 90 - φʳ # Angle between axis of rotation and north pole (degrees)
+ζᵣ(λ, φ, z) = - U * R * (sind(φ) * cosd(α) - cosd(λ) * cosd(φ) * sind(α))
+
 """
     create_test_data(grid, region)
 
 Create an array with integer values of the form, e.g., 541 corresponding to region=5, i=4, j=2.
 If `trailing_zeros > 0` then all values are multiplied with `10trailing_zeros`, e.g., for
 `trailing_zeros = 2` we have that 54100 corresponds to region=5, i=4, j=2.
 """
-function create_test_data(grid, region; trailing_zeros=0)
-    Nx, Ny, Nz = size(grid)
+function create_test_data(grid, region; trailing_zeros=0, solid_body_rotation=false, variable_name="tracer")
```

Comment:
I think it's much much cleaner if we have a completely different function to create the vorticity (which is stream function btw?) for a solid-body rotation. This is very different from the original functionality of `create_test_data` which was simply to fill the interior of fields with integers that correspond to their location in the panel so that we know where they came from when we see them in the halo regions.